src/frontends/onnx/frontend/src/op/com.microsoft/quick_gelu.cpp

```diff
@@ -0,0 +1,58 @@
+#include "core/operator_set.hpp"
+#include "exceptions.hpp"
+#include "openvino/frontend/exception.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/sigmoid.hpp"
+
+
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace com_microsoft {
+namespace opset_1 {
+ov::OutputVector quickgelu(const ov::frontend::onnx::Node& node) {
+    // Original Documentation:
+    // https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#com.microsoft.QuickGelu
+    // Goal: Compute x * Sigmoid(alpha * x)
+
+    const auto inputs = node.get_ov_inputs();
+
+    // Only one input (x) so give a check
+    auto num_inputs = inputs.size();
+    FRONT_END_GENERAL_CHECK(num_inputs == 1,
```

Comment:
Hi, I think we will add some changes, could you also use common::default_op_checks(node, 2); which I had introduced some time ago, for example: https://github.com/openvinotoolkit/openvino/blob/e07546d6414768c7fea3769eef40bcb544e9352a/src/frontends/onnx/frontend/src/op/dequantize_linear.cpp#L205

Reply:
I updated it and applied some additional changes: copyrights and additional include